portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualDefinitions.testcase

```diff
@@ -0,0 +1,63 @@
+@component-name = "portal-analytics-cloud"
+definition {
+
+	property analytics.cloud.enabled = "true";
+	property minimum.slave.ram = "32";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Analytics Cloud";
+
+	setUp {
+
+		// Open Analytics Cloud, login, open faro demo workspace and take the token to connect to DXP
+
+		var analyticsCloudURL = PropsUtil.get("analytics.cloud.url");
+		Navigator.openURL(baseURL = "${analyticsCloudURL}");
+
+		// Take token from Analytics Cloud
+
+		ACLogin.loginAs();
+
+		ACWorkspace.openWorkspace();
```

Comment:
Hi @r4ddek , these calls can be replaced by `ACUtils.launchAC()`;

Reply:
Thanks for the update, Alex! :)